jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -1479,4 +1480,32 @@ public int getWindowYPosition() {
     public void setWindowYPosition(int pos) {
         putInteger("WindowYPosition", pos);
     }
+    
+    
+    /**
+     * Gets the monitor number used when creating a window.
+     *
+     * <p>This setting is used only with LWJGL3, it defines which monitor
+     * to use when creating a OpenGL window.
+     *
+     * @return the desired monitor used when creating a OpenGL window
+     * @see #setMonitor(long)
+     */
+    public int getMonitor() {
+        return getInteger("Monitor");
```

Comment:
It's better to wrap the `"Monitor"` string key in a constant object for future refactoring and testing purposes, you can use the `MonitorInfo` to define default Keys there.